src/app/calculator/compressed-air/air-leak/air-leak.service.ts

```diff
@@ -0,0 +1,627 @@
+      leak.utilityCost = inputCopy.facilityCompressorData.utilityCost;
+      leak.utilityType = inputCopy.facilityCompressorData.utilityType;
+      leak.compressorElectricityData = inputCopy.facilityCompressorData.compressorElectricityData;
+    })
+    let inputArray: Array<AirLeakSurveyData> = this.convertAirleakService.convertInputs(inputCopy.compressedAirLeakSurveyInputVec, settings);
+
+    let baselineLeaks: AirLeakSurveyInput = {
+      compressedAirLeakSurveyInputVec: inputArray
+    };
+    let modificationLeaks: AirLeakSurveyInput = {
+      compressedAirLeakSurveyInputVec: Array<AirLeakSurveyData>()
+    };
+    //  Build baseline / modification leak results
+    let leakResults = Array<AirLeakSurveyResult>();
+    baselineLeaks.compressedAirLeakSurveyInputVec.forEach(leak => {
+      if (!leak.selected) {
+        modificationLeaks.compressedAirLeakSurveyInputVec.push(leak);
+      } 
+      let leakResult: AirLeakSurveyResult = this.standaloneService.airLeakSurvey({compressedAirLeakSurveyInputVec: [leak]});
+      leakResult.name = leak.name;
+      leakResult.leakDescription = leak.leakDescription;
+      leakResult.selected = leak.selected;
+      let converted = this.convertAirleakService.convertResult(leakResult, settings);
+      leakResults.push(converted)
+    });
+
+    // Get cumulative leak results
+    let baselineResults: AirLeakSurveyResult = this.standaloneService.airLeakSurvey(baselineLeaks);
+    let modificationResults: AirLeakSurveyResult = this.standaloneService.airLeakSurvey(modificationLeaks);
+    baselineResults = this.convertAirleakService.convertResult(baselineResults, settings);
+    modificationResults = this.convertAirleakService.convertResult(modificationResults, settings);
+
+    let savings: AirLeakSurveyResult = {
+      totalFlowRate: baselineResults.totalFlowRate - modificationResults.totalFlowRate,
+      annualTotalElectricity: baselineResults.annualTotalElectricity - modificationResults.annualTotalElectricity,
+      annualTotalElectricityCost: baselineResults.annualTotalElectricityCost - modificationResults.annualTotalElectricityCost,
+      annualTotalFlowRate: baselineResults.annualTotalFlowRate - modificationResults.annualTotalFlowRate,
+    }
+
+    let outputs: AirLeakSurveyOutput = {
+      leakResults: leakResults,
+      baselineData: baselineResults,
+      modificationData: modificationResults,
+      savingsData: savings,
+      facilityCompressorData: inputCopy.facilityCompressorData,
+    }
+    return outputs;
+  }
+
+  getExampleFacilityCompressorData(): FacilityCompressorData {
+    let exampleData: FacilityCompressorData = {
+      hoursPerYear: 8760,
+      utilityType: 1,
+      utilityCost: 0.12,
+      compressorElectricityData: {
+        compressorControl: 0,
+        compressorControlAdjustment: .25,
+        compressorSpecificPowerControl: 0,
+        compressorSpecificPower: .16
+      },
+    };
+    return exampleData;
+  }
+  
+  
+  getExample() {
```

Comment:
since this is so big and looks like constants. move this data to a separate .ts file.

also, does this need to be converted if in metric? it may be and I'm not seeing where